js/modules/loki_public_chat_api.js

```diff
@@ -306,17 +307,37 @@ class LokiPublicChannelAPI {
     };
   }
 
+  // get moderation actions
+  async pollForModerators() {
+    try {
+      await this.pollOnceForModerators();
+    } catch (e) {
+      log.warn(`Error while polling for public chat moderators: ${e}`);
+    }
+    if (this.running) {
+      this.timers.channel = setTimeout(() => {
```

Comment:
I don't have the full picture but it seems like you might want to store the timer in `this.timers.moderator` otherwise you overwrite the channel timer id?

Reply:
Good catch